tests/cloner_test.go

```diff
@@ -1021,7 +1021,7 @@ var _ = Describe("all clone tests", func() {
 							Expect(err).ToNot(HaveOccurred())
 							restartCount := cloner.Status.ContainerStatuses[0].RestartCount
 							fmt.Fprintf(GinkgoWriter, "INFO: restart count on clone source pod %s: %d\n", clonerPodName, restartCount)
-							Expect(restartCount).To(BeNumerically("<", 2))
+							Expect(restartCount).To(BeNumerically("<", int32(2)))
```

Comment:
if you look at the failure https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_containerized-data-importer/2701/pull-containerized-data-importer-e2e-ceph/1651557222200643584
the issue here is that 2 is too high for this test to pass
Expected <int32>: 2 **to be <** <int>: 2

We have a PR pending that disables this check:
https://github.com/kubevirt/containerized-data-importer/pull/2647#issuecomment-1512887512

Reply:
The exactly failure I was seeing is that the type didn't match `int` vs `int32` so I just cast it.

Reply:
the matcher should play along with that, the full output looks like:
```
{ Failure /root/go/src/kubevirt.io/containerized-data-importer/tests/cloner_test.go:965
Expected
    <int32>: 2
to be <
    <int>: 2
/root/go/src/kubevirt.io/containerized-data-importer/tests/cloner_test.go:1024}
```